### PR TITLE
Set up basic documents

### PR DESCRIPTION
**Pull request by aburrell**
Added some basic documents needed for the alpha release.  Also made PEP8 changes needed to pass the new style tests.

### CODE REVIEW

File: .zenodo.json [hunk 1]
```diff
@@ -1,13 +1,28 @@
...
       {
-        "affiliation": "U.S. Naval Research Laboratory", 
-        "name": "Burrell, Angeline G.",
-        "orcid": "0000-0001-8875-9326"
+        "orcid": "0000-0001-8875-9326", 
+        "affiliation": "Naval Research Laboratory", 
+        "name": "Angeline G. Burrell"
       },
       {
         "affiliation": "University of Michigan",
```
Comment (aburrell) [inline .zenodo.json:20]: @aaronjridley please add your OrcID.

File: .zenodo.json [hunk 2]
```diff
@@ -1,13 +1,28 @@
...
+        "affiliation": "Naval Research Laboratory", 
+        "name": "Angeline G. Burrell"
       },
       {
         "affiliation": "University of Michigan", 
-        "name": "Ridley, Aaron",
+        "name": "Aaron Ridley"
       },
+      {
+        "affiliation": "University of Michigan",
```
Comment (aburrell) [inline .zenodo.json:25]: @qalshidi please add your OrcID.

**Comment by aaronjridley** [comment]
I added my orcid.
Aaron
Aaron Ridley
Professor
734-764-5727 (tel:734-764-5727)
1416 Space Research Building, Ann Arbor (https://maps.google.com/?q=1416%20Space%20Research%20Building%2C%20Ann%20Arbor)

**Comment by aburrell** [comment]
@aaronjridley it looks like you added your OrcID directly to develop instead of adding it to this branch.  Remember to make changes to the correct branch to avoid causing conflicts in the pull request.

**Comment by aaronjridley** [comment]
Sorry. I have a million things on my to-do list and I tried to do this in a
timely fashion in the time that I have. I tried to do it in the right
place, but there are too many places and too little time.

Writing this as I walk from my car to the office for my next meeting, which
I will be eating lunch at because there is no time for lunch today.

Aaron

**Review by qalshidi — APPROVED** [review]
Great documentation. Very thorough :).

**Pull request merged**